src/js/timingDiagram.js

```diff
@@ -116,7 +116,7 @@ function parseMetadata(data, dataTable, ancestry) {
                 executionCallsCount++;
                 // DA: added the "or now()" safeguard for anything still in Starting or otherwise doesn't have a workflowEnd value, for FireCloud.
                 var endDate = workflowEnd || new Date(Date.now());
-                if(endDate == null) {
+                if(workflowEnd == null) {
```

Comment:
in #1438 ensured that `endDate` was non-null: https://github.com/broadinstitute/firecloud-ui/pull/1438/files#diff-ec6f318f92bf7a1ac08a023a5ba64862R117 which broke this `if` clause. Here, I'm returning the `if` clause to its original intent.